### PR TITLE
update [2]

### PR DESCRIPTION
**Pull request by PliskinDev**

### CODE REVIEW

File: schema.json [hunk 1]
```diff
@@ -67,12 +67,13 @@
 				"Vector3*",
 				"Vector2",
 				"Vehicle", "Vehicle*",
-				"Volume", "Volume*"
+				"Volume", "Volume*",
+				"Weapon", "Weapon*"
```
Comment (TheRouletteBoi) [inline schema.json:73]: Weapon Type should not to a type in native.h. Keep as int

File: schema.json [hunk 2]
```diff
@@ -51,6 +51,7 @@
 				"Any", "Any*",
 				"Blip", "Blip*",
 				"Cam", "Cam*",
+				"Faction", "Faction*",
```
Comment (TheRouletteBoi) [inline schema.json:54]: Remove Faction type and use int. These are not common enough to be made a type.

File: schema.json [hunk 3]
```diff
@@ -56,6 +56,7 @@
 				"Hash", "Hash*",
 				"Iterator", "Iterator*",
 				"IteratorSet", "IteratorSet*",
+				"Journal", "Journal*",
```
Comment (TheRouletteBoi) [inline schema.json:59]: For `Journal` you can use `JournalHandle` because it can be used for a class name. Too common to be using Journal

**Review by TheRouletteBoi — CHANGES_REQUESTED** [review]
Please see my comments and make changes.

**Comment by PliskinDev** [comment]
done

**Comment by TheRouletteBoi** [comment]
> done

json failed validation. Please fix the typo

**Comment by PliskinDev** [comment]
fixed

**Comment by PliskinDev** [comment]
> > done
> 
> json failed validation. Please fix the typo

fixed

**Pull request merged**